partner_risk_insurance/models/credit_policy_company.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class CreditPolicyCompany(models.Model):
+    _name = "credit.policy.company"
```

Comment:
**thought(non-blocking):** wouldn't it be easier to add a `is_credit_policy_company` checkbox to `res.partner` instead?

Reply:
In the future, I will add an image and text into the invoice pdf.
I think this way is easier to manage.

I dont' want to add this fields to the partner because usually only one or two credit companies will be created.